python/paddle/fluid/data_feeder.py

```diff
@@ -67,6 +67,9 @@ def convert_dtype(dtype):
             # however, jointly supporting python2 and python3, (as well as python4 maybe)
             # may still be a long-lasting problem.
             return str(dtype)
+        # NOTE(zhangbo): Now numpy not support bfloat, and paddle use uint16 to carry the data of bfloat16, and there binary is consistent.
```

Comment:
... does not support, and paddle uses unit16 to  represent ..., and there binaries are consistent.

Reply:
Done, tks.